infra/hasura/metadata/build-metadata.sh

```
@@ -77,52 +77,52 @@ copy_with_merge() {
     local tenant="$1"
     echo "Building metadata for tenant: $tenant"
-    
-    
+
     if [ ! -d "$TENANTS_DIR/$tenant" ]; then
         echo "Error: Tenant directory $TENANTS_DIR/$tenant not found!"
         return 1
     fi
-    
 
     rm -rf "$BUILD_DIR/$tenant"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)